talos:
  right_arm_controller:
    type: "talos_control/ArucoController"
    end_effector_link: "wrist_right_ft_tool_link"
    base_link: "dummy_base_link"
    joints:
      - arm_right_1_joint
      - arm_right_2_joint
      - arm_right_3_joint
      - arm_right_4_joint
      - arm_right_5_joint
      - arm_right_6_joint
      - arm_right_7_joint

    goal_tolerance: 0.015
    kp_value: 20
  
  left_arm_controller:
    type: "talos_control/ArucoController"
    end_effector_link: "wrist_left_ft_tool_link"
    base_link: "dummy_base_link"
    joints:
      - arm_left_1_joint
      - arm_left_2_joint
      - arm_left_3_joint
      - arm_left_4_joint
      - arm_left_5_joint
      - arm_left_6_joint
      - arm_left_7_joint

    goal_tolerance: 0.015
    kp_value: 20